youtube_dl/extractor/eroprofile.py

```diff
@@ -80,9 +80,10 @@ def _real_extract(self, url):
         video_url = unescapeHTML(self._search_regex(
             r'<source src="([^"]+)', webpage, 'video url'))
         title = self._html_search_regex(
-            r'Title:</th><td>([^<]+)</td>', webpage, 'title')
+            [r'<h1[^>]*>([^<]+)</h1>', r'Title:</th><td>([^<]+)</td>'],
+            webpage, 'title')
         thumbnail = self._search_regex(
-            r'onclick="showVideoPlayer\(\)"><img src="([^"]+)',
+            [r'<div class="playlistItem current">.*<img src="([^"]+)"', r'onclick="showVideoPlayer\(\)"><img src="([^"]+)'],
```

Comment:
Part in the middle should not be greedy. Also use `_parse_html5_media_entries` as main path.

Reply:
> Part in the middle should not be greedy.

Done, thanks.

> Also use `_parse_html5_media_entries` as main path.

I'm not sure what you mean… but this change doesn't touch the `video_url` so far?